runbook/docs/add-urls-to-monitoring-list-and-database.md

```diff
@@ -0,0 +1,77 @@
+# Add URLs to our monitoring list and database
+*Last updated:* 2020/04/29 
+
+*Last run:* 2020/04/29
+
+## Overview
+Occasionally we need to add new URLs to be monitored. This usually happens when current events prompt use to track URLs that didn't exist before, we realize there are pertinent URLs we haven't been tracking, or if a third-party requests us to track URLs. See this [issue](https://github.com/edgi-govdata-archiving/web-monitoring/issues/151) for an example.
+
+Adding new URLs involves 2 main steps:
+
+* Add URLs Internet Archive Wayback Machine
+* Add URLs to Scanner's database
+
+## Prerequsites
+- Have a clone of [web-monitoring-ops](https://github.com/edgi-govdata-archiving/web-monitoring-ops) repository
+
+## Steps
+
+### **Add URLs to Internet Archive Wayback Machine**
+- Clone [wayback-spn-client](https://github.com/Mr0grog/wayback-spn-client)
+- Create a `.txt` file with the list of new URLs to be tracked
+- From `wayback-spn-client` repo: 
+    - `> node index.js path/to/your/url/list.txt`
+        - This process takes about 30sec per url
+        - Make sure output doesn't contain errors
+
+From `web-monitoring-ops` repo:
```

Comment:
Might be good to clarify that the purpose of this part (as opposed to the above “add URLs to IA Wayback Machine” part) is to make sure that these URLs continue to get captured by Wayback in the future.